pytest-selenium/setup.cfg

```diff
@@ -0,0 +1,3 @@
+[tool:pytest]
+addopts=-vs --tb=long --showlocals --html ui-test.html --self-contained-html
+sensitive_url=^(?:https?\:\/\/)?openstax\.org
```

Comment:
What is this field used for? I don't see it used anywhere directly... maybe it is used by an included python package?

Reply:
These are defaults passed into the pytest command. You run pytest --driver Chrome pytest-selenium/tests and these are inserted in there so you don't have to type them out.

Reply:
ah ok. I was just curious about the purpose of `sensitive_url` but thanks for enumerating the others